### PR TITLE
Remove broken links

### LINKED ISSUE #149: Fix the broken links
I was checking the repository and found some broken links and I would like to delete all the broken links. Should I make a pull request for every broken link? or Should I make only one pull request with all the broken links?

### PR DESCRIPTION
**Pull request by pedabraham**
In this pull request I deleted 15 links, which were unaccessible or reported some kind of error, like 404.

Resolves #149

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,7 +295,6 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 ### Kotlin
 
 - [Awesome Kotlin Newsletter](https://kotlin.libhunt.com/newsletter). A weekly overview of the most popular Kotlin news, articles and libraries.
-- [Kotlin Weekly](http://www.kotlinweekly.net/). The weekly newsletter for Kotlin developers, with articles, libraries, videos and podcasts from the Kotlin environment.
```
Comment (ferhatelmas) [inline README.md:303]: seems this is good

File: README.md [hunk 2]
```diff
@@ -84,7 +84,6 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [Fullweb Weekly](http://fullweb.io/). A weekly newsletter for fullstack web developers. It sends you the top 5 link every week about web frontend and backend, devops and user experience.
 - [Pony Foo Weekly](https://ponyfoo.com/weekly). A weekly newsletter discussing interesting and trending topics around the web platform. [Archive](https://ponyfoo.com/weekly/history).
 - [Dev Awesome](https://devawesome.io/). Get the latest programming news, cool libraries and productivity tips in your inbox every two weeks.
-- [Insnippets](https://insnippets.com). A daily newsletter with TL;DR information about interesting topics around web development, Devops and distributed systems.
```
Comment (ferhatelmas) [inline README.md:87]: seems only certificate expired

File: README.md [hunk 3]
```diff
@@ -336,7 +330,6 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 
 ### NoSQL
 
-- [NoSQL Weekly](https://www.nosqlweekly.com/). A free weekly newsletter featuring curated news, articles, new releases, jobs etc related to NoSQL.
```
Comment (ferhatelmas) [inline README.md:339]: certificate issue, this is good

File: README.md [hunk 4]
```diff
@@ -388,9 +380,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [Data Elixir](https://dataelixir.com/). A weekly newsletter of the best data science news and resources from around the web. [Archive](https://dataelixir.com/newsletters/).
 - [Artificial Intelligence Weekly](http://aiweekly.co/). A weekly collection of the best news and resources on Artificial Intelligence amd Machine Learning.
 - [The Wild Week in AI](https://www.getrevue.co/profile/wildml). A weekly AI & Deep Learning newsletter. [Archive](https://www.getrevue.co/profile/wildml#archive).
-- [Machine Learnings](http://subscribe.machinelearnings.co/). A weekly roundup of ML & AI news.
```
Comment (ferhatelmas) [inline README.md:391]: good too

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
thanks, commented for some since they are alive but have issues with certificate, let's keep them for now.

**Comment by pedabraham** [comment]
hello @ferhatelmas! I restored the links that were incorrectly removed. And I also resolved the merge conflicts.

**Review by ferhatelmas — APPROVED** [review]
LGTM

Thanks for the update.

**Pull request merged**